dockerbuild/Dockerfile-backend

```diff
@@ -1,7 +1,7 @@
-FROM openjdk:8
+FROM openjdk:8 as builder
+
+ARG THRIFT_VERSION=0.10.0
```

Comment:
Ideally you would back out the thrift version changes from here.

Reply:
done